@xen-orchestra/rest-api/src/xoa/xoa.controller.mts

```diff
@@ -43,4 +44,13 @@ export class XoaController extends Controller {
       return dashboard
     }
   }
+
+  @Example(pingResponse)
+  @Get('ping')
+  async ping(): Promise<PingResponse> {
+    return {
+      result: 'pong',
+      timestamp: Date.now(),
+    }
+  }
```

Comment:
Have you tested the endpoint without authentication?
I don't think it works with an unauthenticated user.

You can update `authentication.middleware.mts` to accept a `'none'` security name